base/error.jl

```diff
@@ -67,7 +67,7 @@ end
 
 # convert dual arrays (raw bt buffer, array of GC managed values) to a single
 # array of locations
-function _reformat_bt(bt, bt2)
+function _reformat_bt(bt::Array{<:Any,1}, bt2::Array{<:Any,1})
```

Comment:
I'm pretty sure this is always called on `Array{Ptr{Cvoid},1}` and `Array{Any,1}` exactly.